.coveragerc

```diff
@@ -0,0 +1,2 @@
+[run]
+    omit = tests/*
```

Comment:
Is this similar to `--omit 'venv/*'`? If so can we use the same method for both folders?

Reply:
Alright seems to work as the CLI arg, I'll move it over there